client/app/bundles/comments/components/CommentBox/CommentList/CommentList.spec.jsx

```diff
@@ -28,7 +28,7 @@ describe('CommentList', () => {
       id: 2,
       author: 'Furter',
       text: 'ho',
-    }),
+    })
```

Comment:
OK, this one isnlt failing any more against master, but the other one is.  I've removed this change.